h2o-r/tests/testdir_munging/runit_pivot.R

```diff
@@ -0,0 +1,21 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+test.pivot <- function(){
+    print("step1")
+    df = h2o.createFrame(rows = 1000,
+                         cols=3,
+                         factors=10,
+                         integer_fraction=1.0/3,
+                         categorical_fraction=1.0/3,
+                         missing_fraction=0.0,
+                         seed=123)
+    df$C3 = h2o.abs(df$C3)
+    head(df)
+    df2 = h2o.pivot(df,index="C3",column="C2",value="C1")
+    head(df2)
+    expect_equal(nrow(df2),101)
+    expect_equal(ncol(df2),11)
```

Comment:
I don't think it would bog down the test system. We need to ensure both clients behave the same. The test does not have to take a long time. Rather, it just has to ensure behavior between clients are the same.